spec/helpers/claims/show_helper_spec.rb

```diff
@@ -38,4 +38,50 @@
       end
     end
   end
+
+  describe "#policy_name" do
+    subject(:name) { helper.policy_name(claim) }
+
+    context "with a StudentLoans policy" do
+      let(:policy) { StudentLoans }
+
+      it "returns a text string" do
```

Comment:
You can use the shorthand version for these specs eg. `it { is_expected.to eq "student loan" }`

Reply:
I'm not sure the shorthand version is better 🤷  Happy to go with it if preferred.

Reply:
It's not better, it's shorter. There are a lot of `it "returns a text string" do`.